README.md

```diff
@@ -34,14 +34,27 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+Responsive websites respond to the size of the browser at any given point.  (it is always changing size to fit the sizes of the device it’s on.  Adaptive design only changes when it hits a specific point it’s programmed to – once it hits that point it snaps into place.
+
 2. Describe what it means to be mobile first vs desktop first.
 
+A mobile first designed was originally designed for a mobile screen.  The design then works its way up to function on bigger screens.  Desktop first is the opposite – it’s first design is on bigger screens and then the design works its way down to function on smaller screens. 
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
+It changes the original font (16px) to about 10px.  From there, we are able to change the font using REM knowing that however many rems we set our font size to its result will be that set REM multiplied by 10. (ex 2rem =20px).
+
 4. How would you describe preprocessing to someone new to CSS?
 
+Preprocessing is like a library built into a programming language.  It gives us more tools that we can use to make our code more powerful that isn’t available in the regular language.  It then takes the code that we’ve written using those extra tools and converts it into something that that language can read.
```

Comment:
Be careful with the use of library, Preprocessors are actually a tool. but otherwise great answer!